full-stack-tests/backend/src/perftest/SchemaLoader.test.ts

```diff
@@ -0,0 +1,138 @@
+    return path.join(KnownLocations.nativeAssetsDir, "ECSchemas", domain, schemaFileName);
+  }
+
+  function getSchemaPathFromPackage(packageName: string, schemaFileName: string): string {
+    const schemaFile = path.join(__dirname, "..", "..", "..", "node_modules", "@bentley", packageName, schemaFileName);
+    return schemaFile;
+  }
+
+  async function timeBisSchemasLoading(schemaName: string, schemaPath: string): Promise<void> {
+    const rootSubject = { name: "Performance tests", description: "Performance tests" };
+    const snapshotFile: string = IModelTestUtils.prepareOutputFile("Performance", "Performance.bim");
+    const imodel: StandaloneDb = StandaloneDb.createEmpty(snapshotFile, { rootSubject });
+
+    try {
+      await imodel.importSchemas([schemaPath]);   // will throw an exception if import fails
+    } catch (error) {
+      throw new Error(`Failed to import schema ${schemaPath}`);
+    }
+
+    imodel.saveChanges();
+
+    const startTime: number = new Date().getTime();
+    const schemaResult = imodel.nativeDb.getSchema(schemaName);
+    const endTime: number = new Date().getTime();
+
+    if (schemaResult.error !== undefined) {
+      throw new Error(schemaResult.error.message);
+    }
+
+    if (schemaResult.result === undefined) {
+      throw new Error("Schema does not exists");
+    }
+
+    const elapsedTime = endTime - startTime;
+    reporter.addEntry("SchemaLoaderPerfTest", `Loading schema: ${schemaName}`, "Execution time(ms)", elapsedTime, {});
+    imodel.close();
+  }
+
+  async function memoryBisSchemasLoading(schemaName: string, schemaPath: string): Promise<void> {
+    const rootSubject = { name: "Performance tests", description: "Performance tests" };
+    const snapshotFile: string = IModelTestUtils.prepareOutputFile("Performance", "Performance.bim");
+    const imodel: StandaloneDb = StandaloneDb.createEmpty(snapshotFile, { rootSubject });
+
+    try {
+      await imodel.importSchemas([schemaPath]);   // will throw an exception if import fails
+    } catch (error) {
+      throw new Error(`Failed to import schema ${schemaPath}`);
+    }
+
+    imodel.saveChanges();
+
+    const beforeMemory: NodeJS.MemoryUsage = process.memoryUsage();
+    const schemaResult = imodel.nativeDb.getSchema(schemaName);
+    const afterMemory: NodeJS.MemoryUsage = process.memoryUsage();
+
+    if (schemaResult.error !== undefined) {
+      throw new Error(schemaResult.error.message);
+    }
+
+    if (schemaResult.result === undefined) {
+      throw new Error("Schema does not exists");
+    }
+
+    const memoryUsed = afterMemory.heapUsed - beforeMemory.heapUsed;
+    reporter.addEntry("SchemaLoaderPerfTest", `Loading schema: ${schemaName}`, "Memory used(bytes)", memoryUsed, {});
+    imodel.close();
+  }
+
+  it("Time BisSchemas data read from imodel", async () => {
+    await timeBisSchemasLoading("Units", getSchemaPath("Standard", "Units.01.00.07.ecschema.xml"));
```

Comment:
I suggest importing all schemas into the same iModel then getting them one by one.  You can close/reopen the connection between each schema you open.